### PR TITLE
HEAVY PEARL TODO TIMER THING

### PR DESCRIPTION
**Pull request by Cobble8**
yay hopefully this works idk don't wanna wait 21 hours to test it again but Im pretty confident it works

### CODE REVIEW

**Review by NopoTheGamer — APPROVED** [review]
:+1:

**Review by IRONM00N — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java [hunk 1]
```diff
@@ -367,7 +368,9 @@ public void tick() {
 				//Replaced with for loop because in crystal hollows with events the line it's on can shift.
 				for (String line : lines) {
 					if (line.contains("⏣")) {
-						location = Utils.cleanColour(line).replaceAll("[^A-Za-z0-9() ]", "").trim();
+						String l = Utils.cleanColour(line).replaceAll("[^A-Za-z0-9() ]", "").trim();
+						if(!l.equals(location)) { new ScoreboardLocationChangeListener(location, l); }
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java:372]: im not sure why we use a constructor here.

**Review by romangraef — APPROVED** [review]
aside from the very weird event handling this is fine

File: src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java (hunk 1 above)
Comment (Cobble8) [reply src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java:372]: Idk I'm bad at stuff, you can change it if you want

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java:372]: I dont want to, i just wanted to needlessly complain

**Pull request merged**